draft-kowalik-rpp-architecture-02.adoc

```diff
@@ -325,6 +325,10 @@ textual content in responses to provide human-readable localized responses.
 languages will be defined, preferably using HTTP methods, like OPTIONS or HEAD requests.
 * application/rpp+json media type may support multi-language representations, especially for witing operations involving user provided content. Other media types may have different mechanisms for language representation.
 
+==== Prefer Header for Request Validation
+
+RPP may utilize the HTTP `Prefer` header <<RFC7240>> for signalling the preference for either strict or lenient processing of requests. This allows clients to indicate whether they prefer strict adherence to protocol rules or a more lenient approach that may allow for additional flexibility in processing. The default behavior, without the `Prefer` header, would be to process requests lenient.

```

Comment:
I'm not fully happy with very wide term of "protocol adherence" and "flexibility in processing.
```suggestion
RPP may utilize the HTTP `Prefer` header <<RFC7240>> for signalling the preference for either strict or lenient processing of requests. This allows clients to indicate whether they prefer strict validation of message payloads and rejection of requests with unknown properties or a more lenient approach ignoring unknown properties that may allow for additional flexibility in processing. The default behavior, without the `Prefer` header, would be to process requests lenient.
```